src/main/java/stream/flarebot/flarebot/commands/general/PollCommand.java

```diff
@@ -328,17 +331,17 @@ public String getDescription() {
 
     @Override
     public String getUsage() {
-        return "`{%}poll` - Shows if there are any polls running or not\n" +
-                "`{%}poll help` - Shows this help message\n" +
-                "`{%}poll create <question>` - Creates a poll with a specified question\n" +
-                "`{%}poll list` - Lists the polls for the server\n\n" +
-                "`{%}poll <close/open/remove> <pollID>` - Closes, opens or removes a poll with the specified ID\n" +
-                "`{%}poll set <pollID> colour <colour>` - Sets a colour for a poll with the specified ID\n\n" +
-                "`{%}poll options <pollID> <view/remove> <optionID>` - Views or Removes a specific option from a poll\n" +
-                "`{%}poll options <pollID> add <option>` - Adds an option to a poll\n" +
-                "`{%}poll options <pollID> list` - Lists all the options for a poll\n\n" +
-                "`{%}poll setchannel <pollID/\"all\">` - Sets the poll announcement channel for all polls or a specific poll\n" +
-                "`{%}poll vote [pollID] <optionID>` - Votes for an option on a poll (Poll ID is only needed if the server has more than 1 poll)";
+        return "`{%}poll` - Shows if there are any polls running or not.\n" +
+                "`{%}poll help` - Shows this help message.\n" +
+                "`{%}poll create <question>` - Creates a poll with a specified question.\n" +
+                "`{%}poll list` - Lists the polls for the server.\n\n" +
+                "`{%}poll close|open|remove <pollID>` - Closes, opens or removes a poll with the specified ID.\n" +
+                "`{%}poll set <pollID> colour <colour>` - Sets a colour for a poll with the specified ID.\n\n" +
+                "`{%}poll options <pollID> view|remove <optionID>` - Views or Removes a specific option from a poll.\n" +
+                "`{%}poll options <pollID> add <option>` - Adds an option to a poll.\n" +
+                "`{%}poll options <pollID> list` - Lists all the options for a poll.\n\n" +
+                "`{%}poll setchannel <pollID/\"all\">` - Sets the poll announcement channel for all polls or a specific poll.\n" +
```

Comment:
I don't know how i feel about this

Reply:
Why? It looks fine when it's displayed

Reply:
It doesn't fallow the rest of the usage. we should separate out the setchannel usage

Reply:
OK